src/context.rs

```diff
@@ -127,6 +128,39 @@ fn quote_ident(val: &str) -> String {
     val.replace('"', "\"\"")
 }
 
+pub fn remove_quotes_from_string(possibly_quoted_name: &str) -> String {
+    possibly_quoted_name.trim_matches('"').to_string()
+}
+
+pub fn remove_quotes_from_ident(possibly_quoted_name: &Ident) -> Ident {
+    Ident::new(remove_quotes_from_string(&possibly_quoted_name.value))
+}
+
+pub fn remove_quotes_from_idents(column_names: &[Ident]) -> Vec<Ident> {
+    column_names.iter().map(remove_quotes_from_ident).collect()
+}
+
+pub fn remove_quotes_from_object_name(name: &ObjectName) -> ObjectName {
+    ObjectName(remove_quotes_from_idents(&name.0))
+}
+
+pub fn remove_quotes_from_schema_name(name: &SchemaName) -> SchemaName {
+    match name {
+        SchemaName::Simple(schema_name) => {
+            SchemaName::Simple(remove_quotes_from_object_name(schema_name))
+        }
+        SchemaName::UnnamedAuthorization(auth) => {
+            SchemaName::UnnamedAuthorization(remove_quotes_from_ident(auth))
+        }
+        SchemaName::NamedAuthorization(schema_name, auth) => {
+            SchemaName::NamedAuthorization(
+                remove_quotes_from_object_name(schema_name),
+                remove_quotes_from_ident(auth),
+            )
```

Comment:
Are these ones necessary / used by us?

Reply:
Not that I know of. I thought de-quoting them might be useful to avoid a future footgun, but since this code is hopefully a temporary fix until DF 16, I agree it's better to remove it.